docs/final report/chapters/engineering-reflection.tex

```diff
@@ -1,12 +1,32 @@
 \chapter{Engineering Reflection} %MAX 1 PAGE
 
-Introduce the chapter, explain what will be descibed. 2 PAGES
+In this section we will describe how the engineering process went. We discuss how we have used scrum to develop the product. Furthermore we will discuss the GitHub pull requests and how we ensured that our code had good quality. As last we will discuss the feedback we got from SIG.
```

Comment:
we'll describe. Je probeert geen nadruk op will te leggen, dus samentrekken is mooier.